app/src/main/java/sdp/moneyrun/ui/menu/LeaderboardActivity.java

```diff
@@ -92,7 +92,8 @@ public void addPlayer(Player player){
      *  Initializes the player object private instance
      */
     public void setMainPlayer(Player player){
-        if(user == null && player != null){
+        if(user != null && player != null){
+            System.out.println("Main player is being set");
```

Comment:
Change to a Log ? Or in any case, dont leave Sys out lying around

Reply:
Agreed, will delete this was just for testing purposes. Thanks